packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -16,6 +16,32 @@ export type AwsSdkMetadata = {[key: string]: any};
 
 const awsSdkMetadata: AwsSdkMetadata = metadata;
 
+/**
+ * Physical ID of the custom resource.
+ */
+export class PhysicalResourceId {
+
+  /**
+   * Construct a resource id from the path to the data in the API call response.
+   */
+  public static fromResponsePath(responsePath: string): PhysicalResourceId {
```

Comment:
I think this can be `PhysicalResourceId.fromResponse` (the fact that this is a path is encoded in the argument name)

Reply:
How about `fromResponse(path: string)`? Minimal stutter.

Reply:
Used `fromResponse(responsePath)` so that `responsePath` will appear in the template and make it a bit clearer than just `path`.